pyproject.toml

```diff
@@ -37,4 +37,7 @@ dependencies = [
 "Bug Tracker" = "https://github.com/catalystneuro/tqdm_publisher/issues"
 
 [tool.hatch.version]
-source = "vcs"
\ No newline at end of file
+source = "vcs"
+
+[project.gui-scripts]
+tqdm_publisher-demo-server = "demo.server:main"
```

Comment:
Since by above, the key should be the thing exposed as a terminal command, the mixture of underscores and dashes makes it look a bit awkward

I'd usually expect a pattern such as `tqdm_publisher run_demo_server` or `tqdm_publisher --run_demo_server`, which would involve altering the `__name__ == "main"` part to be modified to accept input arguments. To avoid that, I wonder if `tqdm_publisher.run_demo_server` is allowed?

Reply:
Updated to call a `cli.py` script that parses all of these commands. The documentation has been updated accordingly.